exporter/exporterhelper/factory.go

```diff
@@ -78,6 +79,7 @@ func NewFactory(
 	for _, opt := range options {
 		opt(f)
 	}
+	internal.RegisterMetrics()
```

Comment:
Can you please clarify why this change? Does this add a functionality or it is forced by refactoring?